src/components/pages/Onboarding/Onboarding.tsx

```diff
@@ -182,7 +188,18 @@ class OnboardingView extends React.Component<OnboardingProps, OnboardingState> {
       this.state.verifiedName &&
       this.state.emailAddress
     ) {
-      this._goToStep(this._validatedCameraStep());
+      this._goToStep(this._validatedInviteTokenStep());
```

Comment:
Should this case be removed now? This was for skipping the identity video if the user has a valid video already

Reply:
This is still used to check whether the video for the invite token that the user input exists before advancing.